web/src/pages/billing/BillingHome.tsx

```diff
@@ -68,10 +81,71 @@ const MenuItems = (props: MenuProps) => {
     )
 }
 
+const renderLinkItem = (label: string, href: string) => (
```

Comment:
Both this and `renderGridColumn` are are basically react components so it would be better to make them components rather than functions. The only change needed is to move the arguments into a `props` argument, remove the word `render` from the function name and call them via jsx rather than as a function.